src/plugins/discover/public/application/components/default_discover_table/_table_cell.scss

```diff
@@ -88,7 +88,7 @@ $osdDocTableCellPadding: calc($euiSizeM / 2); // corresponds to DataGrid medium
 // nested for specificity
 .osd-table {
   .osdDocTableCell {
-    padding: $osdDocTableCellPadding;
+    padding: $euiSizeS;
```

Comment:
does this go from 6 -> 8?

Reply:
Checked with UX and Viraj; the change is good to stay.